modules/openapi-generator/src/main/resources/csharp-netcore/ApiClient.mustache

```diff
@@ -271,6 +271,21 @@ namespace {{packageName}}.Client
             {
                 foreach (var headerParam in configuration.DefaultHeaders)
                 {
+                   if (String.Equals(headerParam.Key, "cookie", StringComparison.OrdinalIgnoreCase))
```

Comment:
@vvb thanks for the PR but looks like this line is not indented correctly

Can you also update the petstore samples as well?

Reply:
Thank you for catching this. I have updated the code.